### DISCUSSION TITLE
Prestige IAQ

### DISCUSSION THREAD

**Discussion by sseller** [d0n4v4nb3ck3r/homebridge-honeywell-home#805] [General]
I currently have an ecobee installed. Just had my heater replaced and it included the Honeywell Prestige IAQ. It is pretty cool. Does a lot more the ecobee. I was just wondering if anyone has tried this model with the plugin? I don't use a lot of features in homekit related to the thermostat but having them temp in the home app and that basic control is  nice. Just debating having them back to swap this new one in.

Thanks!

**Comment by donavanbecker** [comment]
Best thing to do is to try it.😉

**Comment by sseller** [comment]
True, true. Only thing is it is not a DIY do/undo so working it in m head 😊
